frontend/src/pages/user/UserAnswerPage.js

```diff
@@ -39,9 +41,36 @@ export default function UserAnswerPage() {
   const lastPage =
     questions && Array.isArray(questions) ? questions.length + 1 : 0;
 
+  const totalQuestion =
+    questions && Array.isArray(questions) ? questions.length : 0;
+  const passingScore = 75;
+
+  const calculateTotalScore = () => {
+    return (score / totalQuestion).toFixed(2) * 100;
+  };
+
+  useEffect(() => {
+    const totalScore = calculateTotalScore();
+
+    setFailed(totalScore < passingScore ? true : false);
+    setPassed(totalScore >= passingScore ? true : false);
```

Comment:
You can just use 1 variable instead of 2.

Reply:
Actually I already tried that sir I get conflicts and bug to my UserAnswerResultPage. And I have no choice but to separate both of them. :bow:

Reply:
@elyric-sun-asterisk-dev i dont get why it has conflict or bug. You have a choice, to fix the bug if it has. So still create 1 variable for this one.